### PR TITLE
feat(Active-Sprint Bug fix):Used currentSprintNumber as a Input data-…

### PR DESCRIPTION
**Pull request by Kanishk6677**
Signed-off-by: Kanishk6677 <kanishkraj6677@gmail.com>

### Functionality:
Fixed the bug regarding Active Sprint button

### Solution:
Used Data binding via teamCurrentSprintNumber in other component

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

File: src/app/body/board/sprint-details/sprint-details.component.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { AngularFireFunctions } from '@angular/fire/functions';
 import { ErrorHandlerService } from 'src/app/services/error-handler.service';
 import { BackendService } from 'src/app/services/backend.service';
 import { ApplicationSettingsService } from 'src/app/services/application-settings.service';
-import { Sprint } from 'src/app/Interface/TeamInterface';
+import { Sprint,SprintDataId,TeamDataId } from 'src/app/Interface/TeamInterface';
```
Comment (shivamgoyal7) [inline src/app/body/board/sprint-details/sprint-details.component.ts:7]: revert this change, this is not required

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]

File: src/app/body/board/board.component.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
                     </ng-container>
                 </div>
                 <ng-container *ngIf="showContent; else loader">
-                    <app-sprint-details [currentSprintName]="currentSprintName" [sprintData]="sprintData" (currentSprint)="changeSprintNumber($event)"></app-sprint-details>
+                    <app-sprint-details [currentSprintName]="currentSprintName" [currentSprintNumber]="teamCurrentSprintNumber"  [sprintData]="sprintData" (currentSprint)="changeSprintNumber($event)"></app-sprint-details>
```
Comment (shivamgoyal7) [inline src/app/body/board/board.component.html:13]: check properly, this is not woking

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
test the code properly, this code is not working properly

**Review by shivamgoyal7 — APPROVED** [review]
ship it

**Review by viv9k — APPROVED** [review]
Ship it

**Comment by k0malSharma** [comment]
Resolve merge conflict asap

**Pull request merged**